### PR TITLE
Add blog for the Kubeflow Community Survey 2021

### PR DESCRIPTION
**Pull request by castrojo**
Submitting this on behalf of @jbottum -- the content here has had multiple reviews in the gdocs, so I am trying to avoid too many edits last minute since we want publish this this week! 

My only concern is how the pictures will turn out, so adding a WIP tag so we can see what the netlify preview looks like.

### CODE REVIEW

**Comment by theadactyl** [comment]
Thanks for this and including me on early reviews! Looks great.
/approve
/lgtm

File: _posts/2021-03-19-kubeflow-continues-to-move-to-production.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+![alt_text](images/2021-03-11-survey/image1.png)
+
+# Kubeflow Continues to Move Into Production
+
+Forty eight percent of users are supporting deployments that are in production, up from 15% last year. Further, one question that many folks have is “do people upgrade a production deployment or just install a new cluster and start over?”. It appears that the latter is far more common: just 8% have upgraded their environment.
+
+![alt_text](images/2021-03-11-survey/image2.png)
+
+Similar to previous years, Kubeflow Pipelines and Notebooks are the most popular components, but other components are now being widely deployed as well. Interest in TensorBoard has grown, joining KFServing, Katib (Hyperparameter Tuning), and Distributed Training as top additional services.
```
Comment (andreyvelich) [inline _posts/2021-03-19-kubeflow-continues-to-move-to-production.md:26]: I think we can name Katib is an AutoML system to be more precise.
```suggestion
Similar to previous years, Kubeflow Pipelines and Notebooks are the most popular components, but other components are now being widely deployed as well. Interest in TensorBoard has grown, joining KFServing, Katib (AutoML), and Distributed Training as top additional services.
```

File: _posts/2021-03-19-kubeflow-continues-to-move-to-production.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+*   Updated and more in-depth documentation
+
+# Key Takeaways 
+
+In addition to learning about how users are operating Kubeflow in production clusters, the Community Survey has given us important data that we can use to enhance our processes: 
+
+*   Improvements to the release management process, which is being driven by better inter-Working Group collaboration. This, coupled with core upgrades to Istio and a clean-up of the installation manifests, will improve the testing, documentation, and installation patterns.
+*   Automation of pipeline building and feature engineering tasks, especially with continued integrations of Kubeflow with Kale and Feast, which are enabling new end-to-end workflows and tutorials.
+*   Enhancements for data preprocessing and transformation as well as streamlined connections of Data Pipelines to ML Pipelines.  Additionally, a renewed effort to develop a Spark operator.
+*   Several new UIs are under development i.e. Hyperparameter Tuning, Model Management, Volumes Management and TensorBoard Management, which will help the user experience.
```
Comment (andreyvelich) [inline _posts/2021-03-19-kubeflow-continues-to-move-to-production.md:96]:
```suggestion
*   Several new UIs are under development i.e. Katib, Model Management, Volumes Management and TensorBoard Management, which will help the user experience.
```

**Review by andreyvelich — COMMENTED** [review]
Thank you for the great work @jbottum @castrojo!
I left few comments.

File: _posts/2021-03-19-kubeflow-continues-to-move-to-production.md (hunk 2 above)
Comment (castrojo) [reply _posts/2021-03-19-kubeflow-continues-to-move-to-production.md]: Fiixed in both places, thanks!

**Pull request merged**